dali/ft/fttransform.cpp

```diff
@@ -577,7 +577,7 @@ void TransferServer::deserializeAction(MemoryBuffer & msg, unsigned action)
 {
     SocketEndpoint ep;
     ep.deserialize(msg);
-    if (!ep.isLocal())
+    if (!isContainerized() && !ep.isLocal())
```

Comment:
Why isn't #ifndef _CONTAINERIZED used here?

Reply:
because it is more verbose, and prevents the code being syntax checked in a containerized build.